includes/class-analytics.php

```diff
@@ -271,5 +285,82 @@ protected static function output_js_scroll_event( $event ) {
 		</script>
 		<?php
 	}
+
+	/**
+	 * Output JS for a form submit-based event listener.
+	 *
+	 * @param array $event Event info. See 'get_events'.
+	 */
+	protected static function output_js_submit_event( $event ) {
+		?>
+		<script>
+			( function() {
+				var elementSelector = '<?php echo esc_attr( $event['element'] ); ?>';
+				var elements        = Array.prototype.slice.call( document.querySelectorAll( elementSelector ) );
+
+				for ( var i = 0; i < elements.length; ++i ) {
+					elements[i].addEventListener( 'submit', function() {
+						gtag(
+							'event',
+							'<?php echo esc_attr( $event['event_name'] ); ?>',
+							{
+								event_category: '<?php echo esc_attr( $event['event_category'] ); ?>',
+								event_label: '<?php echo esc_attr( $event['event_label'] ); ?>',
+							}
+						);
+					} );
+				}
+			} )();
+		</script>
+		<?php
+	}
+
+	/**
+	 * Output JS for a load event listener.
+	 *
+	 * @param array $event Event info. See 'get_events'.
+	 */
+	protected static function output_js_ini_load_event( $event ) {
+		$element = isset( $event['element'] ) ? $event['element'] : '';
+		?>
+		<script>
+			( function() {
+				var handleEvent = function() {
+					gtag(
+						'event',
+						'<?php echo esc_attr( $event['event_name'] ); ?>',
+						{
+							event_category: '<?php echo esc_attr( $event['event_category'] ); ?>',
+							event_label: '<?php echo esc_attr( $event['event_label'] ); ?>',
```

Comment:
This should be a non interactive event, otherwise GA bounce rate will be inaccurate (it will be 0).

Reply:
Good catch, thanks! Fixed in ff75db5